src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -99,6 +104,17 @@ public static Builder builder(Class<?> type) {
     return builder(ClassName.get(type));
   }
 
+  public static AnnotationSpec get(AnnotationMirror annotation) {
```

Comment:
nit: do you mind moving this above the `builder()` methods? Awkward to separate those from the `Builder` itself